src/core/showElement.js

```diff
@@ -458,6 +458,16 @@ export default function _showElement(targetElement) {
     _disableInteraction.call(self);
   }
 
+  // hide skip button
+  if (this._options.hideSkip === true) {
```

Comment:
Could you add a test case for this if-statement please?

Reply:
@afshinm added two test cases, please have a look.